backend/src/main/java/com/relay/controllers/RelayController.java

```diff
@@ -126,7 +124,7 @@ public Relay createRelay(@Valid @RequestBody Relay relay) {
         // URI uri = ServletUriComponentsBuilder.fromCurrentRequest().path("/{id}")
         // .buildAndExpand(savedRelay.getId()).toUri();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)